### PR TITLE
networkpools support table

### PR DESCRIPTION
**Pull request by paurosello**
Add support table

### CODE REVIEW

**Comment by calvix** [comment]
how does it look?

**Review by calvix — APPROVED** [review]

**Comment by paurosello** [comment]
This is not the "alpha labels support" it's just the widget you can see on https://docs.giantswarm.io/basics/nodepools/ to let users know when it was introduced

**Comment by calvix** [comment]
ack

File: src/content/basics/networkpools/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Network Pools
 description: A general description of network pools as a concept, it's benefits, and some details you should be aware of.
-date: 2021-01-09
```
Comment (marians) [inline src/content/basics/networkpools/index.md:4]: Please remove the date field. This is no longer needed.

**Review by marians — APPROVED** [review]

File: src/content/basics/networkpools/index.md (hunk 1 above)
Comment (paurosello) [reply src/content/basics/networkpools/index.md:4]: Done!

**Comment by marians** [comment]
@calvix Here is how it looks:

image

https://docs.giantswarm.io/basics/networkpools/

This is something we introduced specifically based on your feedback, if I remember correctly.

**Pull request merged**